.travis.yml

```diff
@@ -1,20 +1,25 @@
 language: python
 sudo: required
+cache: pip
 services:
 - docker
 python:
-- 3.4
+- 3.6
```

Comment:
What is the lowest version of python we want to support for r2d? What ever that version is I think we should add a build matrix step for that. For me 3.6 is fine, how exclusionary do we think is requiring 3.6, especially because we tell people to "run r2d locally to debug your binder problems".

Reply:
Good point. Probably 3.4 or 3.5

Reply:
added a test to run the venv builds with 3.4